cmd/kubeadm/app/cmd/token.go

```diff
@@ -296,27 +296,29 @@ func RunListTokens(out io.Writer, errW io.Writer, client clientset.Interface) er
 	return nil
 }
 
-// RunDeleteToken removes a bootstrap token from the server.
-func RunDeleteToken(out io.Writer, client clientset.Interface, tokenIDOrToken string) error {
-	// Assume the given first argument is a token id and try to parse it
-	tokenID := tokenIDOrToken
-	klog.V(1).Infoln("[token] parsing token ID")
-	if !bootstraputil.IsValidBootstrapTokenID(tokenIDOrToken) {
-		// Okay, the full token with both id and secret was probably passed. Parse it and extract the ID only
-		bts, err := kubeadmapiv1beta1.NewBootstrapTokenString(tokenIDOrToken)
-		if err != nil {
-			return errors.Errorf("given token or token id %q didn't match pattern %q or %q",
-				tokenIDOrToken, bootstrapapi.BootstrapTokenIDPattern, bootstrapapi.BootstrapTokenIDPattern)
+// RunDeleteTokens removes a bootstrap tokens from the server.
+func RunDeleteTokens(out io.Writer, client clientset.Interface, tokenIDsOrTokens []string) error {
+	for _, tokenIDOrToken := range tokenIDsOrTokens {
+		// Assume the given first argument is a token id and try to parse it
+		tokenID := tokenIDOrToken
+		klog.V(1).Infoln("[token] parsing token ID")
+		if !bootstraputil.IsValidBootstrapTokenID(tokenIDOrToken) {
+			// Okay, the full token with both id and secret was probably passed. Parse it and extract the ID only
+			bts, err := kubeadmapiv1beta1.NewBootstrapTokenString(tokenIDOrToken)
+			if err != nil {
+				return errors.Errorf("given token or token id %q didn't match pattern %q or %q",
+					tokenIDOrToken, bootstrapapi.BootstrapTokenIDPattern, bootstrapapi.BootstrapTokenIDPattern)
+			}
+			tokenID = bts.ID
 		}
-		tokenID = bts.ID
-	}
 
-	tokenSecretName := bootstraputil.BootstrapTokenSecretName(tokenID)
-	klog.V(1).Infoln("[token] deleting token")
-	if err := client.CoreV1().Secrets(metav1.NamespaceSystem).Delete(tokenSecretName, nil); err != nil {
-		return errors.Wrap(err, "failed to delete bootstrap token")
+		tokenSecretName := bootstraputil.BootstrapTokenSecretName(tokenID)
+		klog.V(1).Infoln("[token] deleting token")
```

Comment:
Can you print the token id here?

Reply:
I agree with @rosti here. I'll remove this line.